tfjs-backend-webgl/src/flags_webgl.ts

```diff
@@ -208,3 +208,11 @@ ENV.registerFlag(
             `manual flush) or at least 0, but got ${threshold}.`);
       }
     });
+
+/**
+ * Threshold for input tensor size that determines whether WebGL backend will
+ * delegate computation to CPU.
+ *
+ * Default value is 128.
+ */
+ENV.registerFlag('CPU_HANDOFF_SIZE_THRESHOLD', () => 128);
```

Comment:
@pyu10055 Should this flag be registered in tfjs-core flags instead of flags_webgl.ts ? Or we should have this flag both in webgl and webgpu? Otherwise, if weblg is not registered, will `CPU_HANDOFF_SIZE_THRESHOLD` be always undefined for webgpu?

Reply:
Hi @qjia7, WebGPU should also have this flag. It's GPU-specific flag, so shouldn't be in core's flag.

Reply:
Sounds good for me. Add PR #5119